base/_fonts.scss

```diff
@@ -0,0 +1,39 @@
+// -----------------------------------------------------------------------------
+// This file contains all @font-face declarations, if any.
+// -----------------------------------------------------------------------------
+
+
+// Google Fonts
+// -------------------------------------------------------
+// Font weights and variants:
+// Extra-Light: 200; Light: 300; Regular: 400; Semi-Bold: 600; Bold: 700; Extra Bold: 800; Black: 900
+// Italic variants: 200i, 300i, etc.
+// Full font families:
+// $google-source-code: 'https://fonts.googleapis.com/css?family=Source+Code+Pro';
+// $google-source-serif: 'https://fonts.googleapis.com/css?family=Source+Serif+Pro:400,600,700';
+// $google-source-sans: 'https://fonts.googleapis.com/css?family=Source+Sans+Pro:200,200i,300,300i,400,400i,600,600i,700,700i,900,900i';
+// $google-noto-sans: 'https://fonts.googleapis.com/css?family=Noto+Sans:400,400i,700,700i';
+// $google-noto-serif: 'https://fonts.googleapis.com/css?family=Noto+Serif:400,400i,700,700i';
+// Conditional Declarations
+// -------------------------------------------------------
+
+@if(global-variable-exists(source-code)) {
```

Comment:
I wonder if it would make sense to separate these in to separate files and create a function/mixin that @imports the correct files. I did a quick google search for font loading patterns and I did not get much back. I'll add an issue to track this idea for a later version.